### PR TITLE
Ferrari Line Follower and Obstacle Avoider Car 

### PR DESCRIPTION
**Pull request by IonescuMariaMagdalena**
### Pull Request Overview

This pull request fixes the documentation

### Testing Strategy

This pull request was tested by...

### TODO or Help Wanted

This pull request still needs to be reviewed

### Build

- [ ] Ran `npm build`.

### Author

Signed-off-by: Ionescu Maria-Magdalena <maria.ionescu0511@stud.fils.upb.ro>

### CODE REVIEW

File: docs/project/2024/maria.ionescu/index.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+By engaging in this project, I want to explore the frontiers of autonomous robotics and contribute to the advancement of technology in meaningful and impactful ways. Also, I've always been a huge fan of cars, especially Ferraris and it's a chance to show my DIY skills and dive into the world of robotics and electronics. 
+
+## Architecture 
+
+The architecture should sort of look like this. I will be using a white cardbox with a black line for the path and different objects as obstacles. 
+This project is a smart car using a Raspberry Pi Pico W and various sensors and components. It has the following strucure:
+
+- The Raspberry Pi Pico W is a microcontroller board based on the RP2040 microcontroller chip developed by Raspberry Pi, that manages the project's operations, controlling the servo motor, processing sensor data, and handling other components.
```
Comment (alexandruradovici) [inline docs/project/2024/maria.ionescu/index.md:24]:
```suggestion
 - The Raspberry Pi Pico W is a microcontroller board based on the RP2040 microcontroller chip developed by Raspberry Pi, that manages the project's operations, controlling the servo motor, processing sensor data, and handling other components.
```
Indent one for a list item.

File: docs/project/2024/maria.ionescu/index.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+Pololu QTRX-HD-06A Reflective IR Sensor
+L298N Motor Driver 
+Mini Breadboard
+Active Buzzer Module
+Male-female, male-male, female-female wires
+Switch l Insulating electrical tape
+Double-sided adhesive white tape
+Connectors and screws
+Chassis
+Pin headers
```
Comment (alexandruradovici) [inline docs/project/2024/maria.ionescu/index.md:60]: These will be all displayed on a single line.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Comment by alexandruradovici** [comment]
Why a new PR?

**Comment by IonescuMariaMagdalena** [comment]
I updated the index.md. By mistake, I've submitted a new pull request, I apologize

**Review by alexandruradovici — APPROVED** [review]

File: docs/project/2024/maria.ionescu/architecture.jpg
Comment (alexandruradovici) [inline docs/project/2024/maria.ionescu/architecture.jpg:1]: Please use the images in the *index.md* file.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/maria.ionescu/index.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
+# Ferrari Line Follower and Obstacle Avoider Car 
+
```
Comment (alexandruradovici) [inline docs/project/2024/maria.ionescu/index.md:2]: Please add a one line description of your project,

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Review by alexandruradovici — APPROVED** [review]

**Review by irina-nita — APPROVED** [review]

**Pull request merged**